src/app/server/AppDelegate.h

```diff
@@ -28,6 +28,9 @@ class AppDelegate
     virtual ~AppDelegate() {}
     virtual void OnCommissioningSessionStarted() {}
     virtual void OnCommissioningSessionStopped() {}
+
+    // Use the helper function CommissioningWindowStatus
```

Comment:
Please use /** style comments.

Also, would be good to provide a description of what the function does ("This is called anytime a basic or enhanced commissioning window is opened. The type of the window can be retrieved by calling CommissioningWindowManager::CommissioningWindowStatus()").

Reply:
Updated! :+1: